CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 -
 
 ### Changed
--
+- **capital-framework:** [PATCH] Standardized all the usage docs for v4
```

Comment:
I duplicated what was used for the `4.0.1` release, but it would be good to be certain.